desci-server/prisma/migrations/20231128184308_add_draft_node_tree_table/migration.sql

```diff
@@ -0,0 +1,20 @@
+-- CreateTable
+CREATE TABLE "DraftNodeTree" (
+    "id" SERIAL NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
+    "path" TEXT,
+    "cid" TEXT NOT NULL,
+    "directory" BOOLEAN NOT NULL,
+    "size" INTEGER NOT NULL,
+    "nodeId" INTEGER NOT NULL,
+    "userId" INTEGER NOT NULL,
+
+    CONSTRAINT "DraftNodeTree_pkey" PRIMARY KEY ("id")
+);
+
```

Comment:
Curious to know if you considered having a `children` column/relation instead of paths, and how that'd compare :eyes: 

I imagined each file and dir being unaware of its parental structure, but listing its children. But maybe this would lead to unknown-depth recursion hard to do in a single query, could that be the reason?